app/controllers/licence_controller.rb

```diff
@@ -14,101 +14,64 @@ class LicenceController < ApplicationController
   NO_MATCHING_AUTHORITY = 'noLaMatch'.freeze
   NO_MAPIT_MATCH = 'fullPostcodeNoMapitMatch'.freeze
 
-  # NOTE: This is a temporary fix to ensure that these licences get treated as
-  # 'county/unitary' tiered (as opposed to 'district/unitary'). The tier data
-  # for licences used to be stored in LocalService model records in the content
-  # api, but the entries for the licences below have since been removed. In
-  # future this tier data will be stored in the licensing application
-  # (Licensify). Once that has happened and Frontend has been updated to use
-  # that tier information, this list and related code in the
-  # `local_authority_slug` method can be removed
-  LICENCE_SLUGS_WITH_COUNTY_TIER_OVERRIDE = [
-    'scaffolding-and-hoarding-licence',
-    'skip-operator-licence',
-    'permission-to-place-tables-and-chairs-on-the-pavement',
-    'pavement-or-street-display-licence',
-    'petroleum-storage-licence',
-    'weighbridge-operator-certificate',
-    'performing-animals-registration',
-    'approval-of-premises-for-civil-marriage-or-civil-partnership',
-    'licence-projection-over-highway-england-wales',
-  ].freeze
-
-  def search
-    @interaction_details = licence_details
-
-    if request.post?
-      if postcode_search_submitted?
-        @postcode = postcode
-        @location_error = location_error
-
-        if local_authority_slug
-          redirect_to licence_authority_path(slug: params[:slug], authority_slug: local_authority_slug)
-        end
-      end
-    elsif @publication.continuation_link.present?
+  def start
+    if @publication.continuation_link.present?
```

Comment:
This method is super hard to follow. in part because it's actually two controller actions mashed together. 

Might it be easier to read if there's a single top-level conditional that differentiates between the `GET` and `POST` handlers? 

I know that this leads to some potential duplication, but IMO it's preferable to have duplicated code than complicated logic.